docs/docs/60-new-docs/50-user-guide/60-reference-docs/50-analysis-templates.md

```diff
@@ -0,0 +1,216 @@
+---
+sidebar_label: Analysis Templates
+description: Learn about AnalysisTemplate for verification
+---
+
+# Analysis Template Reference
+
+An `AnalysisTemplate` is a resource that defines how to perform verification testing, including:
+* container images and commands to run
+* queries to external monitoring tools
+* how to interpret results from metric providers
+* success or failure criteria
+* frequency and duration of measurements
+
+`AnalysisTemplate` resources (and the `AnalysisRun` resources that are spawned from them) are CRDs re-used from the [Argo Rollouts](https://argoproj.github.io/argo-rollouts) project. They were intentionally built to be useful in contexts other than Argo Rollouts. Re-using this resource type to define verification processes means those processes benefit from this rich and battle-tested feature of Argo Rollouts.
+
+:::info
+This reference guide is intended to give a brief introduction to AnalysisTemplates for some common use cases. Please consult the [relevant sections](https://argoproj.github.io/argo-rollouts/features/analysis/) of the Argo Rollouts documentation for comprehensive coverage of the full range of `AnalysisTemplate` capabilities.
+:::
+
+AnalysisTemplates integrate natively with many popular open-source and commercial monitoring tools, including:
+* [Prometheus](https://prometheus.io/)
+* [DataDog](https://www.datadoghq.com/)
+* [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/)
+* [NewRelic](https://newrelic.com/)
+* [InfluxDB](https://www.influxdata.com/)
+* [Apache SkyWalking](https://skywalking.apache.org/)
+* [Graphite](https://graphiteapp.org/)
+
+In addition to monitoring tools, analysis can integrate with internal systems by:
+* Running containerized process as a Kubernetes Job
+* Making HTTP requests to URLs and interpreting JSON responses
+
+
+## Arguments
+
+AnalysisTemplates may declare a set of arguments that can be "passed" in by the `Stage`. The arguments are resolved at the time the AnalysisRun is created and can then be referenced in metrics configuration. Arguments are dereferenced using the syntax: `{{ args.<name> }}`.
```

Comment:
```suggestion
`AnalysisTemplate`s may declare a set of arguments that can be "passed" in by the `Stage`. The arguments are resolved at the time the `AnalysisRun` is created and can then be referenced in metrics configuration. Arguments are dereferenced using the syntax: `{{ args.<name> }}`.
```

Reply:
#3408 is adding expression language support here, so we'll have to add a note later about `{{ }}` vs `${{ }}`.